org.zdoom.GZDoom.yaml

```diff
@@ -0,0 +1,108 @@
+app-id: org.zdoom.GZDoom
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: "3.32"
+command: gzdoom.sh
+rename-icon: gzdoom
+
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --share=ipc
+- --socket=pulseaudio
+
+# Where would wads be?
+- --filesystem=xdg-desktop:ro
+- --filesystem=xdg-documents:ro
+- --filesystem=xdg-download:ro
+
+# Screenshots? 
+# Sceenshots go by default in ~/.config/
+#- --filesystem=xdg-pictures
+
+# Based on GZDoom
+# We access the original ~/.config/gzdoom because
+# that is where users should put their iwads
+- --env=DOOMWADDIR=/app/share/games/doom
+- --filesystem=~/.config/gzdoom
```

Comment:
Not sure if was this discussed for other games, can't we have it use the xdg-data directory?

Reply:
Problem is, that GZDoom doesn't actually have any game dta files, so users are asked to provide their own. I'm not sure what is the best way in letting users to add their own proprietary data. The current implementation of GZDoom does ask the user to add them to ~/.config/GZDoom, so I decided to stick with that for now.

I did play around with a _persist_ but it only made it more confusing

Reply:
For clarity, technically it's completely possible. It's more about consistency to the front-end user. This is the most likely place where they already have all their game data.